### PR TITLE
Translate new strings into Vietnamese

### PR DESCRIPTION
**Pull request by cuongdcdev**
Translated **exportAccounts** section and make some adjustments

### CODE REVIEW

**Comment by cuongdcdev** [comment]
cc @DamirSQA 👀

**Review by DamirSQA — APPROVED** [review]

**Comment by DamirSQA** [comment]
Hi @cuongdcdev , thanks for the update.

**Comment by DamirSQA** [comment]
a

File: packages/core/src/lib/locale/vi.json [hunk 1]
```diff
@@ -48,12 +48,57 @@
...
+      }
+    },
+    "component": {
+      "clickToCopy": {
+        "label": "Đã sao chép",
+        "tooltip": "Bấm để sao chép"
       }
     }
-  }
+  }
```
Comment (DamirSQA) [inline packages/core/src/lib/locale/vi.json:104]: s

**Pull request merged**